cmd/meroxa/root/apps/apps.go

```diff
@@ -30,6 +31,34 @@ import (
 	"github.com/spf13/cobra"
 )
 
+type ApplicationState string
+
+const (
+	ApplicationStateInitialized ApplicationState = "initialized"
+	ApplicationStateDeploying   ApplicationState = "deploying"
+	ApplicationStatePending     ApplicationState = "pending"
+	ApplicationStateRunning     ApplicationState = "running"
+	ApplicationStateDegraded    ApplicationState = "degraded"
+	ApplicationStateFailed      ApplicationState = "failed"
+
+	// collectionName = "apps".
```

Comment:
I assume this is not needed? Maybe we could remove in that case.

Reply:
This is not needed in this pr, i'll remove it for now!